### ISSUE TITLE
SRTM Europe not available

### ISSUE THREAD

**Issue by RafaelAbel** [firelab/windninja#395]
Dear WindNinja Team,
for both the GUI and the cli the European SRTM Data are no longer reachable.

ERROR 4: `/vsizip//vsicurl/http://dds.cr.usgs.gov/srtm/version2_1/SRTM3/Eurasia/N48E009.hgt.zip/N48E009.hgt' does not exist in the file system, and is not recognized as a supported dataset name.

There is also a 30m resolution available which you might want to implement.
https://e4ftl01.cr.usgs.gov/MEASURES/SRTMGL1.003/

Best,

Rafael

**Comment by nwagenbrenner** [comment]
@RafaelAbel Thanks for reporting this. It looks like there was a recent change to the way the USGS data are distributed. This affects our SRTM and Landfire data. We're working on a fix. Thanks also for including the information about the 30 m global SRTM data as well, we'll look into incorporating that.

**Comment by nws-brockburghardt** [comment]
Hi @nwagenbrenner noticed the same thing in WindNinja when trying to retrieve 30m and 90m SRTM data. Any idea on where new USGS https path is? or timeline on a fix within WindNinja? Thanks -Brock

**Comment by nwagenbrenner** [comment]
@nws-brockburghardt We're working on a fix, but it's not a simple path change, it's a significant change to the data request protocol and will likely take some time to get a fix implemented. Until then, the GMTED DEM source is the alternative option if you want to have WindNinja download your DEM. We'll try to get a fix out as soon as we can.

**Comment by RafaelAbel** [comment]
Hi @nwagenbrenner, any updates from your side on that issue?

**Comment by nwagenbrenner** [comment]
@RafaelAbel Sorry, we still don't have a fix for this yet, but we hope to get to it soon.

**Comment by nwagenbrenner** [comment]
It looks like we can now get the 30 m Global SRTM data from the OpenTopography API. We meet their terms of service. We should implement this.

https://portal.opentopography.org/apidocs/#/Public/getGlobalDem

Requests are like:

```
curl -X 'GET' \
  'https://portal.opentopography.org/API/globaldem?demtype=SRTMGL1&south=50&north=50.1&west=14.35&east=14.6&outputFormat=GTiff&API_Key=XXXX' \
  -H 'accept: */*'
```

**Comment by nwagenbrenner** [comment]
Fixed in 15f905c316f1d0a4b037249af0fa6a46b20b98dc through ef049e2756e12e84ad1ffb06029cdd844701049f. We need to update the tutorials, but otherwise this is ready to go. We've implemented the GL1 SRTM dataset (Global SRTM at 30 m) via the OpenTopography API. We should also consider adding the 30 m COP30 global dataset. Apparently this is now considered to be the best available global dataset by many metrics.
